frontend/src/container/OnboardingContainer/Modules/APM/Java/md-docs/Others/Kubernetes/others-kubernetes-runApplication.md

```diff
@@ -3,10 +3,12 @@
 Once you are done intrumenting your Java application, you can run it using the below command
 
 ```bash
-java -javaagent:<path>/opentelemetry-javaagent.jar -jar {{MYAPP}}.jar
+java -javaagent:<path>/opentelemetry-javaagent.jar -jar <my-app>.jar
```

Comment:
This value is replaced by the value entered by the user in the service name input box. We don't need it now?

Reply:
This is actually the name of the app jar file that the user creates, which can be different from the service name. 
That's why removed {{MYAPP}}